api-tests/features/p2-strategic-app.feature

```diff
@@ -59,7 +59,11 @@ Feature: M2B Strategic App Journeys
     Then I get a 'pyi-triage-select-device' page response
     When I submit a 'smartphone' event
     Then I get a 'pyi-triage-select-smartphone' page response
-    When I submit an 'end' event
+    When I submit a 'neither' event
+    Then I get a 'pyi-triage-select-smartphone' page response
+    When I submit an 'neither' event
+    Then I get a 'pyi-triage-buffer' page response
+    When I submit an 'anotherWay' event
```

Comment:
I don't think this is quite right now this page has been updated - https://identity.build.account.gov.uk/dev/template/pyi-triage-select-smartphone/en?context=mam. The 'No' option according to the designs should now go to the DAD select smartphone page

Reply:
I'm not sure if we've captured this so we might be best raising a separate ticket for it (and just leave it how it was for now)

Reply:
Ah expanding this I can see this is exactly what this does, sorry 😄 
Can we add the page contexts now for `pyi-triage-select-smartphone`?

Reply:
I'll merge in main and see what needs to change